docs/server.rst

```diff
@@ -184,3 +184,45 @@ After you configured Tunneldigger, you can run the broker::
 
     cd /srv/tunneldigger
     /srv/env_tunneldigger/bin/python -m tunneldigger_broker.main /srv/tunneldigger/broker/l2tp_broker.cfg
+
+Docker Image
+------------
+
+There is a docker image available from `wlanslovenija/tunneldigger-broker
+<https://hub.docker.com/r/wlanslovenija/tunneldigger-broker>`_.
```

Comment:
I am not sure if I am comfortable providing "official" docker images. Isn't the `docker build` below enough?

Disclaimer: I have very little docker experience, and the experience I have consists mostly of fights and frustration about how much disk space it is using.

Reply:
I can not answer if you are comfortable with this.
I added a TODO.

> create automated build or delete the section about providing the image

I would be able to configure the automated build as well as integrate it into the travis build. However, as this image is not of any use at this point in time, I do not work on it now.

Reply:
Yeah, I think the image is more useful as "machine-readable installation instructions" than for direct (re-)use. So it probably makes more sense not to have it on the hub.

Reply:
I would create a new issue for hosting it. For me, it would be useful as I can build another image on top of it. However, I will create the issue when I succeed with this image.